Aptoide/aptoide-client-v8
# [refactor/ASV-1294] Increase SDK version

**What does this PR do?**

   This essentially increases the targetSdkVersion to 27 and aims to fix any problems that occured. Specifically, additional manifest permissions were added and notification channels were added.

   Right now, the code for notification channels is being repeated. We could extract it to an utility class or similar and inject it, but I don't know if it's worth it.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] InstallService.java
- [ ] PullingContentService.java

**How should this be manually tested?**

  Everything needs to be tested to check if it's working correctly, since the targetSdkVersion change might introduce several bugs. Notifications should specifically be checked since there's known changes there.

**What are the relevant tickets?**

  [ASV-1294](https://aptoide.atlassian.net/browse/ASV-1294)

**Code Review Checklist**
